examples/flow_tool/test.cc

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2018 Dmitry "Dima" Korolev <dmitry.korolev@gmail.com>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*******************************************************************************/
+
+#define CURRENT_MOCK_TIME
+
+#include "flow_tool.h"
+
+#include "../../bricks/dflags/dflags.h"
+#include "../../bricks/file/file.h"
+
+#include "../../3rdparty/gtest/gtest-main-with-dflags.h"
+
+#ifndef CURRENT_WINDOWS
+DEFINE_string(flow_tool_test_tmpdir, ".current", "Local path for the test to create temporary files in.");
+#else
+DEFINE_string(flow_tool_test_tmpdir, ".", "Local path for the test to create temporary files in.");
```

Comment:
Why different per platform? A comment could help.

Reply:
Because Studio doesn't use Makefiles. I copied it from another example, thought of removing everything non-*nix, and then thought: why bother? :)

Reply:
How is this related to Makefiles?

Reply:
The only entity that creates the '.current` directory is our Makefile, which is unused by Visual Studio.